st2client/st2client/commands/resource.py

```diff
@@ -735,16 +736,29 @@ def __init__(self, resource, *args, **kwargs):
     def run(self, args, **kwargs):
         resource_id = getattr(args, self.pk_argument_name, None)
         instance = self.get_resource(resource_id, **kwargs)
-        self.manager.delete(instance, **kwargs)
+        if isinstance(instance, st2client.models.action.Action):
```

Comment:
Is it possible to have a cli argument like --yes so that it is possible to delete actions within having to give input. This makes it much easier for anyone using st2 cli in scripts.

Reply:
I agree, giving the option `-y` or `--yes` makes this scriptable.

Reply:
Also, you should move this changes to the ActionDeleteCommand at https://github.com/StackStorm/st2/blob/master/st2client/st2client/commands/action.py#L280 instead of using if isinstance to check if this resource is an Action.

Reply:
@amanda11 Sure, I will try to add functionality to have CLI argument --yes. Thanks.